### ISSUE TITLE
Partitioned DML over 30s always fails by context deadline exceeded

### ISSUE THREAD

**Issue by apstndb** [cloudspannerecosystem/spanner-cli#102]
Partitioned DML over 30s always fails by context deadline exceeded.

```
$ time ./spanner-cli -p ${CLOUDSDK_CORE_PROJECT} -i ${CLOUDSDK_SPANNER_INSTANCE} -d ${DATABASE_ID} --execute='PARTITIONED UPDATE Songs SET Duration = Duration + 1 WHERE TRUE' 
ERROR: rpc error: code = DeadlineExceeded desc = context deadline exceeded
./spanner-cli -p ${CLOUDSDK_CORE_PROJECT} -i ${CLOUDSDK_SPANNER_INSTANCE} -d   0.35s user 0.47s system 2% cpu 34.068 total
```

`gcloud databases execute-sql` succeeds.

```
$ time gcloud spanner databases execute-sql --instance=merpay-sponsored-instance ${DATABASE_ID} --sql='UPDATE Songs SET Duration = Duration + 1 WHERE TRUE' --enable-partitioned-dml
Statement modified a lower bound of 1024000 rows
gcloud spanner databases execute-sql --instance=merpay-sponsored-instance     0.55s user 0.22s system 1% cpu 1:10.73 total
```

**Comment by yfuruyama** [comment]
Thanks for filing an issue. Let me check if this happens on my side as well.

**Comment by yfuruyama** [comment]
I found the reason why this happens.

* spanner-cli calls google-cloud-go/spanner's `PartitionedUpdate` function: [statement.go](https://github.com/cloudspannerecosystem/spanner-cli/blob/4fbafa7c6bf8232cba454bbee19b6d01181922d2/statement.go#L771)
* `PartitionUpdate` function internally calls `spanner.Client.ExecuteSql` function: [pdml.go](https://github.com/googleapis/google-cloud-go/blob/spanner/v1.11.0/spanner/pdml.go#L116)
* `spanner.Client.ExecuteSql` set timeout with 30 seconds if the context doesn't have any timeout: [spanner_client.go](https://github.com/googleapis/google-cloud-go/blob/spanner/v1.11.0/spanner/apiv1/spanner_client.go#L318-L322)

So basically we have to set longer timeout explicitly to run PDML for over 30 secs. I will create a PR to fix that.
